libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackClientWrapper.cs

```diff
@@ -45,20 +46,20 @@ public SlackClientWrapper(SlackAdapterOptions options)
                                        "****************************************************************************************" +
                                        ">> Slack docs: https://api.slack.com/docs/verifying-requests-from-slack";
 
-                throw new Exception(warning + Environment.NewLine + "Required: include a verificationToken or clientSigningSecret to verify incoming Events API webhooks");
+                throw new InvalidOperationException(warning + Environment.NewLine + "Required: include a verificationToken or clientSigningSecret to verify incoming Events API webhooks");
```

Comment:
It may be better to say ERROR instead of Warning since we are throwing an exception, not a biggie just consider it.